src/lib/ui/device/BatteryStatus.tsx

```diff
@@ -12,21 +12,29 @@ interface BatteryStatusProps {
 export function BatteryStatus(props: BatteryStatusProps): JSX.Element {
   return (
     <div className='seam-battery-status'>
-      <Content status={props.device.properties.battery?.status} />
+      <Content
+        status={props.device.properties.battery?.status}
+        level={props.device.properties.battery?.level}
+      />
     </div>
   )
 }
 
 function Content(props: {
   status: SeamBatteryStatus | null | undefined
+  level: number | null | undefined
 }): JSX.Element | null {
-  const { status } = props
+  const { status, level } = props
+
+  const percentage = level ? ` (${Math.floor(level * 100)}%)` : null
```

Comment:
The linter error is protecting us from rendering null instead of 0 in the 0 level case.

```suggestion
  const percentage = level == null ? ` (${Math.floor(level * 100)}%)` : null
```